### PR TITLE
able tò scape Ambitionbox  website and get compaines with there ratings

### PR DESCRIPTION
**Pull request by PlanetDestroyyer**
## Description

> The changes made in this pull request should be briefly described.

## Checklist

> Before submitting this pull request, kindly verify that the ensuing checkpoints have been reached.

- [x] Have you adhered to the repository's defined coding convention rules?
- [x] Have you updated the 'documentation.md' file with the method/function documentation?
- [ ] Have you sent a message along with the result or response?
- [ ] Have you used the try-catch technique?
- [x] Has the method/class been added to the documentation (md file)?

## Screenshots

> Uploading a screenshot illustrating the approach you developed for validation is required.

## Additional Notes/Comments

> Any additional remarks or suggestions concerning this pull request are welcome.

---

I certify that I have carried out the relevant checks and provided the requisite screenshot for validation by submitting this pull request.
I appreciate your contribution.

### CODE REVIEW

File: documentation.md [hunk 1]
```diff
@@ -733,3 +733,25 @@ boxoffice = imdb.BoxOffice()
 | Methods         | Details                                                                       |
 | --------------- | ----------------------------------------------------------------------------- |
 | `.top_movies()` | Returns the top box office movies, weekend and total gross and weeks released |
+
```
Comment (nikhil25803) [inline documentation.md:736]: Add the documentation part in `dev-documentation.md` not in the `documentation.md` file.

**Review by nikhil25803 — CHANGES_REQUESTED** [review]

**Comment by PlanetDestroyyer** [comment]
@nikhil25803 i have added changes can review it once

**Review by nikhil25803 — APPROVED** [review]

**Comment by nikhil25803** [comment]
Great work @PlanetDestroyyer  !!
Do not forget to
+ Share your contribution on socials and tag Clueless 🫂
+ Give this repository a star ✨
+ Follow Clueless Community on GitHub ✅

**Comment by PlanetDestroyyer** [comment]
Thank you @nikhil25803

**Pull request merged**